src/openforms/forms/api/serializers/form.py

```diff
@@ -109,6 +109,7 @@ class FormSerializer(PublicFieldsSerializerMixin, serializers.ModelSerializer):
     )
     is_deleted = serializers.BooleanField(source="_is_deleted", required=False)
     required_fields_with_asterisk = serializers.SerializerMethodField(read_only=True)
+    resume_link_lifetime = serializers.SerializerMethodField(read_only=True)
```

Comment:
place provide a `label` and `description`, such as "this is an integer expressing the number of days before..."